servers/audio/audio_stream.cpp

```diff
@@ -105,6 +105,30 @@ void AudioStreamPlaybackResampled::_begin_resample() {
 	mix_offset = 0;
 }
 
+int AudioStreamPlaybackResampled::_mix_internal(AudioFrame *p_buffer, int p_frames) {
+	int ret;
+	if (GDVIRTUAL_CALL(_mix_resampled, p_buffer, p_frames, ret)) {
+		return ret;
+	}
+	WARN_PRINT_ONCE("AudioStreamResampled::_mix_resampled unimplemented!");
+	return 0;
+}
+float AudioStreamPlaybackResampled::get_stream_sampling_rate() {
+	float ret;
+	if (GDVIRTUAL_CALL(_get_stream_sampling_rate, ret)) {
+		return ret;
+	}
+	WARN_PRINT_ONCE("AudioStreamResampled::_mix_resampled unimplemented!");
```

Comment:
```suggestion
	WARN_PRINT_ONCE("AudioStreamResampled::_get_stream_sampling_rate unimplemented!");
```

Though maybe this was the first one you changed before settling on `ERR_FAIL_V_MSG(0, "Unimplemented function");`?